### ISSUE TITLE
Feature Request: Add Accounts

### ISSUE THREAD

**Issue by raymonddurk** [BlueWallet/BlueWallet#1989]
I'd love for the ability to add accounts to the wallet. Right now when you add another wallet, it literally adds another wallet which means a secondary, tertiary, etc. seed to save, store, and keep track of. I would love the ability to add accounts the way Green Wallet, Trezor, and others do within the same seed. This makes it easier to manage funds in the absence of things like coin control as it is achieved on a wallet basis as opposed to a UTXO basis.

**Comment by pointbiz** [comment]
Coin control now exists in Bluewallet. Do you still want this feature? 

I'm against this feature. Mycelium works that way and it makes managing multiple accounts confusing because they are all tied to one seed.

If you have multiple wallets you should assume there's different seed requirements (owner, storage, etc.) for the wallet therefore each account should be a wallet with a proper seed. Using a  seed for every account (wallet) has the advantage of not needing to expose an xpriv in the UI.

**Comment by Overtorment** [comment]
we might add it as a "compatibility" feature during the import

**Comment by raymonddurk** [comment]
Coin control is the more granular way of doing it for things like "Mike, Exchange, coinjoin change, etc" but there are certain times where there are themes where I can add an account for, "Fold, Lolli, Exchange, etc" where I don't need to coin control each one of those. The account name is the coin control. Other things need actual coin control but it depends on how each user manages their wallets.

**Comment by Transisto** [comment]
I agree a feature like this might be needed, IMO It's the simplest implementation of coin-control that most people can understand.
